python/lsst/afw/image/wcs.i

```diff
@@ -48,39 +48,20 @@
 %lsst_persistable(lsst::afw::image::Wcs);
 %lsst_persistable(lsst::afw::image::TanWcs);
 
-%inline %{
-    #include <memory>
-    #include <boost/serialization/serialization.hpp>
-    #include <boost/archive/binary_oarchive.hpp>
-    #include <boost/archive/binary_iarchive.hpp>
-    #include "lsst/daf/persistence/PropertySetFormatter.h"
-    #include <sstream>
-    std::string pickleMetadata(CONST_PTR(lsst::daf::base::PropertySet) header) {
-        std::stringstream ss;
-        boost::archive::binary_oarchive ar(ss);
-        ar << *header;
-        return ss.str();
-    }
-    PTR(lsst::daf::base::PropertySet) unpickleMetadata(std::string const& pick) {
-        std::stringstream ss(pick);
-        boost::archive::binary_iarchive ar(ss);
-        PTR(lsst::daf::base::PropertySet) header = std::make_shared<lsst::daf::base::PropertySet>();
-        ar >> *header;
-        return header;
-    }
-%}
-
 %pythoncode %{
     def unpickleWcs(pick):
-        header = unpickleMetadata(pick)
-        return makeWcs(header)
+        import pickle
```

Comment:
`cPickle` does not exist on python 3. Are you asking for us to explicitly attempt to load `cPickle` for python2 and `pickle` on python3? What's the default protocol. I don't think @PaulPrice cares.